run_pr.sh

```diff
@@ -20,7 +20,7 @@ if [ $? != 0 ]; then exit 1; fi
 # 2) KITCHEN_YAML
 # 3) KITCHEN_GLOBAL_YAML (lowest)
 export KITCHEN_LOCAL_YAML=$WORKSPACE/repo/.kitchen-override.yml
-export KITCHEN_YAML=~jenkins/.kitchen/config.yml
+export KITCHEN_YAML=~jenkins/.kitchen/config_sdc.yml
```

Comment:
yes, this file already exists.  When I manually edited the run_pr.sh to look like this and then kicked off the job the same way jenkins does it actually built on SDC